pumps/mongo.go

```diff
@@ -552,21 +552,17 @@ func (m *MongoPump) AccumulateSet(data []interface{}, isForGraphRecords bool) []
 		}
 
 		// Skip this record if it is a graph analytics record, they will be handled in a different pump
-		if thisItem.IsGraphRecord() != isForGraphRecords {
+		isGraphRecord := thisItem.IsGraphRecord()
+		if isGraphRecord != isForGraphRecords {
 			continue
 		}
-		if isForGraphRecords {
-			if thisItem.RawRequest == "" || thisItem.RawResponse == "" || thisItem.ApiSchema == "" {
-				continue
-			}
-		}
```

Comment:
Why did we remove this block? Does it break the `max-document-size-bytes` logic?

Reply:
No it's an unneeded check, we should be discarding records that do not have those, that logic was initially there thinking the rawrequest had to absolutely be available, but now, we've realized it's possible to have records without them, in fact it's more than likely...The graph pump would throw a warning anyway when it encounters such records, it's better to be logged, than silently discarded